pages/reference/connection-strings.mdx

```diff
@@ -0,0 +1,24 @@
+# Connection Strings
+
+In order to connect to your source database, Snaplet needs read-only database credentials in the form of a connection string, or connection URL. 
+
+A PostgreSQL connection URL specifies the following parameters:
+
+* **username** (the username used to connect to the database)
+* **password** (the password used in the "user" parameter)
+* **hostname** (the IP address or domain name of the machine where the server is running)
+* **port** (port number on which the server is listening on)
+* **database** (name of the database to connect to)
+
+This connection string may also include a collection of optional parameter keywords that allow adjustments of various aspects of the URL (e.g. SSL, timeouts, etc).
+
+Here's an example connection string:
+
+```postgresql://username:password@hostname:5432/database_name```
+
+### Troubleshooting
+When passing in a connection string to Snaplet, we will attempt to validate and encode the connection string, however you may still have problems with database names or passwords with special characters (`%&/:=?@[]`), you may have to encode these characters manually.
+
+Encoding manually: you can manually encode individual segments of your connection url (password, database, etc.) using [encodeURIComponent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)
+
+If you have trouble configuring your connection string, you can to refer to this [Prisma guide](https://www.prisma.io/dataguide/postgresql/short-guides/connection-uris) or reach out to us on [Discord](https://app.snaplet.dev/chat) instead.
```

Comment:
Ported over and cleaned up from the old docs.

Reply:
I would add that as a section in the PostgreSQL guide, it doesn't deserve its own page to me.

Reply:
Thanks - will pick this up in a future PR.